### DISCUSSION TITLE
Replace the dashboard with a custom page?

### DISCUSSION THREAD

**Discussion by limegorilla** [keystonejs/keystone#7899] [Questions]
I would like to be able to re-create the dashboard and pull in a load of extra data - think like analytics info, account info etc. and I would like this to be the main page people see when they login.

I can create this page fine inside custom pages using client-side API calls, but I cannot seem to be able to replace the main page in any way (setting the file to be `admin/pages/index.tsx` didn't work)

**Comment by limegorilla** [comment]
Weird - this is now working.

I am getting a warning:
```
warn  - Duplicate page detected. pages/index.tsx and pages/index.js both resolve to /index.
```

Only thing I can think of is that `tsx` files are preferred over `js`?

**Reply by flexdinesh** [reply]
Default Admin UI pages are generated using templates. In dev environment, when you write your own page, it replaces the default page that is generated from templates. There isn’t a way to stop generating the pages from templates at the moment and the warning is expected and safe to ignore.
